models/basket.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  const basket = sequelize.define('basket', {
+    clientId: {
+      type: DataTypes.INTEGER,
+      allowNull: false
+      
+    },
+
+  }, {});
+  basket.associate = function(models) {
+    basket.belongsTo(models.client, { foreignKey: 'clientId'})
+
+    basket.belongsToMany(models.product, { through: 'basketProducts',
+                                            unique: false});
+   
+  };
+  return basket;
+};
```

Comment:
another detail, this weird symbol here is because JS files are supposed to have an empty line at the end